openai_vector/challenges/run-benchmark.json

```diff
@@ -0,0 +1,67 @@
+{
+    "name": "run-benchmark",
+    "default": true,
+    "description": "Challenge to close, restore, and benchmark index with specified operations.",
+    "schedule": [
+      {
+        "name": "close-benchmark-index",
+        "operation": {
+          "operation-type": "raw-request",
+          "path": "/vdb910-openai/_close",
+          "method": "POST"
+        }
+      },
+      {
+        "name": "restore-snapshot",
+        "operation": {
+          "operation-type": "raw-request",
+          "path": "/_snapshot/vdb910-openai/vdb910-openai/_restore",
+          "method": "POST",
+          "body": {
+            "indices": "vdb910-openai"
+          }
+        }
+      },
+      {
+        "name": "check-cluster-health",
+        "operation": "check-cluster-health"
+      },
+      {
+        "name": "wait-until-merges-finish-after-index",
```

Comment:
Since we have closed the index and the only indices we have access to is vdb910-openai, the GET request returns below response
`{"_shards":{"total":0,"successful":0,"failed":0},"_all":{"primaries":{},"total":{}},"indices":{}}`
this doesn't match the condition